library/src/amd_detail/rocblaslt/src/tensile_host.cpp

```diff
@@ -262,6 +262,20 @@ namespace
                    : typeB;
     }
 
+    rocblaslt_status hip2RocStatus(hipError_t status)
+    {
+        switch(status)
+        {
+        case hipSuccess:
+            return rocblaslt_status_success;
+        case hipErrorUnknown:
+        case hipErrorRuntimeOther:
```

Comment:
The error code of the ticket is hipErrorInvalidDevice. Sould we return rocblaslt_status_internal_error but rather than rocblaslt_status_not_implemented?